samples/ChatbotExample/README.md

```diff
@@ -1,13 +1,13 @@
-﻿# Cognitive Service Personalization
+﻿# Personalizer
 
-This sample shows how Cognitive Service Personalization can be integrated with a chat bot.
+This sample shows how Personalizer can be integrated with a chat bot.
 
-## Run Personalization Chatbot
+## Run Personalizer Chatbot
 
 In order to run this example:
 
 1. Read through [README-LuisBot.md](./README-LuisBot.md) to setup LuisBot
-2. In [RLFeatures.cs](./ReinforcementLearning/RLFeatures.cs), modify HostName to a deployed Personalization Service
+2. In [RLFeatures.cs](./ReinforcementLearning/RLFeatures.cs), modify HostName to a deployed Personalizer service; in [RLContextManager.cs](./ReinforcementLearning/RLContextManager.cs), modify SubscriptionKey to that service's
```

Comment:
Putting keys directly in the code is not recommended since it can easily be accidentally checked in. Reference samples/HttpRequestFeatures or demos/PersonalizerDemo for possible improvements.

Reply:
additionally, since this is a peronalizer sample, maybe consider throwing some argument exceptions if personalizer is not set up correctly.